scripts/install_st2.sh

```diff
@@ -65,7 +65,7 @@ if [[ -n "$RHTEST" ]]; then
     sudo ln -s /opt/rh/python27/root/usr/bin/python /usr/local/bin/python
     sudo ln -s /opt/rh/python27/root/usr/bin/pip /usr/local/bin/pip
     source /etc/environment
-  elif [[ "$RHMAJVER" == '7' ]]; then
+  elif [[ "$RHMAJVER" == '7' || "$RHMAJORVER" == '8' ]]; then
     sudo yum install -y epel-release
     sudo yum install -y python
```

Comment:
So - EL8 requires `numbered` python, so if you want python2 or python3 install a number is needed. 

```suggestion
  elif [[ "$RHMAJVER" == '7' ]]; then
    sudo yum install -y epel-release
    sudo yum install -y python
  elif [[ "$RHMAJVER" == '8' ]]; then
    sudo yum install -y epel-release
    sudo yum install -y python2 # python3
```

Reply:
Fixed.